### PR TITLE
Updated audit log information to syslog format

### PR DESCRIPTION
**Pull request by sunithasomanna**
Updated audit log information to syslog format in API readme and added a new section "Application logs" and explained each property.

### CODE REVIEW

**Review by ashraf-vazeer — APPROVED** [review]
LGTM

File: docs/README.md [hunk 1]
```diff
@@ -12312,17 +12312,29 @@ curl -i -X POST \
...
+Audit logs provide information on each API and are stored in the `api.log` file in `odimra` logs. 
 
-**Sample logs**
-
-- <110> 2009-11-10T23:00:00Z xxx.xxx.xxx.xxx [account@1 user="admin" roleID="Administrator"][request@1 method="GET" resource="/redfish/v1/Systems" requestBody=""][response@1 responseCode=200] Operation Successful
+**Sample log**
 
-- <110> 2022-01-17T13:57:48Z xxx.xxx.xxx.xxx [account@1 user="admin" roleID="Administrator"][request@1 method="POST" resource="/redfish/v1/AggregationService/AggregationSources" requestBody="{"HostName":"xxx.xxx.xxx.xxx","Links":{"ConnectionMethod":{"@odata.id":"/redfish/v1/AggregationService/ConnectionMethods/337ea3cb-3acc-49e2-b33f-3f5ce2a5ada4"}},"Password":"null","UserName":"admin"}"][response@1 responseCode=202] Operation successful
+```
+<14> 1 2022-11-29T10:39:23Z odim-controller svc-systems systems-744b9475c6-bgvwm_7 GetSystemCollection [process@1 processName="systems-744b9475c6-bgvwm" transactionID="e77fe640-d8d5-4c2e-a219-31b217fee0da" actionID="001" actionName="GetSystemCollection" threadID="0" threadName="svc-systems"] Inside GetSystemsCollection function (GetResource)
```
Comment (amar-shalgar) [inline docs/README.md]: This is not audit log

File: docs/README.md [hunk 2]
```diff
@@ -12312,17 +12312,29 @@ curl -i -X POST \
 
 # Audit logs
 
-Audit logs provide information on each API and are stored in the `api.log` file in `odimra` logs.  Each log consists of a priority value, date and time of the log, hostname from which the APIs are sent, user account and role details, API request method and resource, response body, response code, and the message.
```
Comment (amar-shalgar) [inline docs/README.md:12315]: This info still holds good

File: docs/README.md [hunk 3]
```diff
@@ -12312,17 +12312,29 @@ curl -i -X POST \
...
+```
+<14> 1 2022-11-29T10:39:23Z odim-controller svc-systems systems-744b9475c6-bgvwm_7 GetSystemCollection [process@1 processName="systems-744b9475c6-bgvwm" transactionID="e77fe640-d8d5-4c2e-a219-31b217fee0da" actionID="001" actionName="GetSystemCollection" threadID="0" threadName="svc-systems"] Inside GetSystemsCollection function (GetResource)
+```
 
-- <107> 2009-11-10T23:00:00Z xxx.xxx.xxx.xxx [account@1 user="admin" roleID="Administrator"][request@1 method="GET" resource="/redfish/v1/Systems" requestBody=""][response@1 responseCode=404] Operation failed
+Each log consists of the following properties:
 
-  <blockquote> Note: <110> and <107> are priority values. <110> is the audit information log and <107> is the audit error log. </blockquote>
+- **<14>**, **<110>** or **<107>**—Priority values. 
+  <110> is the value for successful operation, <107> is the value for failed operation. <14> is the value for log information.
```
Comment (amar-shalgar) [inline docs/README.md]: 110 and 107 are for audit logs success and error logs. there is a big list of priority values , pls add all

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: fixed

Comment (sunithasomanna) [reply docs/README.md]: retained the existing content of audit log

File: docs/README.md (hunk 3 above)
Comment (sunithasomanna) [reply docs/README.md]: referred users to priority values in Log levels section in Getting Started Readme

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md:12315]: retained the existing content of audit log

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**